build_node/builders/base_rpm_builder.py

```diff
@@ -139,8 +139,9 @@ def build(self):
                 finally:
                     os.chdir(cwd)
             else:
-                source_srpm_dir = self.unpack_sources()
-                spec_file = self.locate_spec_file(source_srpm_dir)
+                source_srpm_dir = self.unpack_sources(True) # download only
+                spec_file = None
```

Comment:
I guess it's unnecessary to set spec_file to None because we already do it on 89 line

Reply:
Indeed.